TicTacToe/MinimaxAlgorithm.fs

```diff
@@ -1,14 +1,37 @@
 ﻿module MinimaxAlgorithm
 open GameBoard
+open Game
 
-let score gameState player1 player2 boardSize=
-    if didPlayer1Win gameState player1 boardSize then
-        10
-    elif didPlayer2Win gameState player2 boardSize then
-        -10
+let rec minimaxAlgorithm (game : Game, isComputerPlayer) =
+
+    if(didPlayer1Win game.CurrentBoard game.ComputerCharacter game.BoardSize) then
+        [0;1]
+    elif(didPlayer2Win game.CurrentBoard game.HumanCharacter game.BoardSize) then
+        [0;-1]
+    elif(didTieHappen game.CurrentBoard) then
+        [0;0]
     else
-        0
+        if isComputerPlayer then
```

Comment:
Each branch of this `if` statement looks identical except for the `true`/`false` in the recursive call. Is there a way to condense this down?